src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -18,18 +18,10 @@
 import time
 from abc import ABC, abstractmethod
 from argparse import ArgumentParser, _ArgumentGroup
+from collections.abc import Iterator
 from contextlib import contextmanager
 from threading import Condition
-from typing import (Any,
-                    ContextManager,
-                    Dict,
-                    Iterator,
-                    List,
-                    NamedTuple,
-                    Optional,
-                    Set,
-                    Union,
-                    cast)
+from typing import Any, ContextManager, NamedTuple, Optional, Union, cast
```

Comment:
Don't we want these imports that change constantly like this to be the multi-line form?

Reply:
This is `black`s doing, from `make format`